openandroidinstaller/tooling.py

```diff
@@ -292,7 +292,12 @@ def adb_twrp_wipe_and_install(
             # if not an a/b-device just stay in twrp
             pass
     else:
-        for line in adb_reboot(bin_path=bin_path):
+        #recovery might not be installed yet , we reboot to fastboot to flash it
```

Comment:
Here a `adb_reboot_recovery` would be needed.

But why only flash it afterwards? If flashing twrp directly is save, we could do it directly and run `fastboot_flash_recovery` to begin with. (like it is done for example for Xiaomi recoveries)